kfs-core/src/main/java/edu/arizona/kfs/sys/KFSKeyConstants.java

```diff
@@ -3,5 +3,9 @@
 public class KFSKeyConstants extends org.kuali.kfs.sys.KFSKeyConstants {
 
     public static final String MESSAGE_DV_DUPLICATE_INVOICE = "message.dv.duplicate.invoice";
-
+    
+    public static final String ERROR_GEC_REF_NUMBER_INVALID = "error.gec.ref.number";

```

Comment:
And again here, good job on obliterating magic strings... only forgot to reference them once added.